docs/current_docs/api/clients-cli.mdx

```diff
@@ -1,17 +1,47 @@
 ---
 slug: /api/cli
 ---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
 # Dagger CLI
 
-The Dagger CLI lets you call both the core and extended Dagger API (the core APIs plus the new APIs provided by Dagger Functions) directly from the command-line via the `dagger core` and `dagger call` commands.
+The Dagger CLI lets you call both the core and extended Dagger API (the core APIs plus the new APIs provided by external Dagger modules) directly from the command-line.
 
-## Core Dagger API functions
+You can call the API interactively (`dagger`) or non-interactively (`dagger -c`, `dagger call`, or `dagger core`).
 
-The `dagger core` command lets you call the core Dagger API, without needing to first load a Dagger [module](../features/modules.mdx). This is useful in some scenarios:
+Here are a few examples:
 
-- You want to use Dagger for simple pipelines that are fully satisfied by the core Dagger API.
+- Create a simple pipeline that is fully satisfied by the core Dagger API, without needing to program a Dagger module:
 
+    <Tabs groupId="shell">
+    <TabItem value="System shell">
+    ```shell
+    dagger <<EOF
+    container |
+      from cgr.dev/chainguard/wolfi-base |
+      with-exec apk add go |
+      with-directory /src https://github.com/golang/example#master |
+      with-workdir /src/hello |
+      with-exec -- go build -o hello . |
+      file ./hello |
+      export ./hello-from-dagger
+    EOF
+    ```
+    </TabItem>
+    <TabItem value="Dagger Shell">
+    ```shell title="Type 'dagger' to start Dagger Shell in interactive mode and enter the command below."
+    container \
+      | from cgr.dev/chainguard/wolfi-base \
+      | with-exec apk add go \
+      | with-directory /src https://github.com/golang/example#master \
+      | with-workdir /src/hello \
+      | with-exec -- go build -o hello . \
+      | file ./hello \
+      | export ./hello-from-dagger
+    ```
+    </TabItem>
+    <TabItem value="Dagger CLI">
```

Comment:
Technically the Dagger Shell is a part of the Dagger CLI.

Reply:
This was answered in https://github.com/dagger/dagger/pull/9905#discussion_r2005008758. Let's move on and update these everywhere when we have a better fit.